src/Component/Router.purs

```diff
@@ -58,7 +57,7 @@ component =
     Profile _ -> HH.div_ []
     Favorites _ -> HH.div_ []
   
-  eval :: Query ~> H.ParentDSL State Query (Const Void) Void Void m
+  eval :: Query ~> H.ParentDSL State Query Home.Query Unit Void m
   eval = case _ of
     Click a -> do
```

Comment:
This can be deleted altogether -- I was using it to test out different API calls briefly.

Reply:
We will eventually need a type to contain all the child component queries. Probably some kind of coproduct

Reply:
Sorry -- I meant the `Click` query in the router. You're right, we'll end up with a fairly large coproduct with each of the pages as a member. There's a nice syntax for when you have a large number of these:

- https://pursuit.purescript.org/packages/purescript-either/4.1.0/docs/Data.Either.Nested#t:type%20(\/)
- https://pursuit.purescript.org/packages/purescript-functors/3.1.0/docs/Data.Functor.Product.Nested#t:type%20(%3C/\%3E)